katello.gemspec

```diff
@@ -56,7 +56,7 @@ Gem::Specification.new do |gem|
   gem.add_dependency "pulp_file_client", ">= 1.2.0", "< 1.6.0"
   gem.add_dependency "pulp_ansible_client", ">= 0.2", "< 0.7"
   gem.add_dependency "pulp_container_client", ">= 2.0.0", "< 2.3.0"
-  gem.add_dependency "pulp_deb_client", ">= 2.6.0", "< 2.9.0"
+  gem.add_dependency "pulp_deb_client", ">= 2.9.0", "< 2.10.0"
```

Comment:
I don't actually know what API bindings compatibility across these versions looks like.

I aim to update the plugin from pulp_deb 2.8.0 to 2.9.1, I don't know why the pulp_deb_client was previously declared `>= 2.6.0`. Foreman-packaging for the `rpm/2.4` branch, contains the 2.8.0 client already.

Reply:
in 4.0 we're shipping 2.8 bindings  today, so i wouldn't worry about 2.6 and 2.7